install_wfa.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Build WFA
+if [ ! -d wfa-source ]; then
+  git clone https://git.firesponse.es/external/vestec.git wfa-source
+  cd wfa-source
+  git checkout 8fce0a845836968e2658a8c64c58f39fba77c652
+  cd ..
+  mkdir wfa-build
+fi
+
+cd wfa-build
+cmake -DUSE_CATALYST=ON \
```

Comment:
This is the CMake variable that will trigger Catalyst adaptor build in the simulation code